tests/test_base_template.py

```diff
@@ -0,0 +1,46 @@
+"""Test the templatekit.repo.BaseTemplate class.
+"""
+
+import os
+import pytest
+from templatekit.repo import BaseTemplate
+
+
+def test_validation():
+    """Test basic BaseTemplate instantiation/validation checks.
+
+    Uses the actual templates repository data.
+    """
+    file_template_exists = BaseTemplate('file_templates/license_gplv3')
```

Comment:
Is it dangerous that these assume the test will be run from the root level of the package?  From experience, I know that I sometimes run from inside `tests`.